scss/_variables.scss

```diff
@@ -423,6 +423,7 @@ $label-margin-bottom:                   .5rem !default;
 $input-padding-y:                       $input-btn-padding-y !default;
 $input-padding-x:                       $input-btn-padding-x !default;
 $input-font-size:                       $input-btn-font-size !default;
+$input-font-weight:                     $font-weight-normal !default;
```

Comment:
Maybe we can use `$font-weight-base` for this? This font weight is also used by `body`?